timetable/templates/substitutions.html

```diff
@@ -28,7 +28,7 @@ <h2 class="text-center">{% trans "Substitutions" %}</h3>
 	{% with group=substitution.lesson.group %}
 	<tr {% if group in groups or timetable_teacher != None and timetable_teacher == substitution.substitute %}class="relevant"{% endif %}>
 		<td>{{ substitution.lesson.period }}</td>
-		<td>{{ substitution.period_str }}</td>
+		<td {% if substitution.substitute == None %} style="text-decoration: line-through;"{% endif %}>{{ substitution.period_str }}</td>
```

Comment:
The case for issue #4 is showing periods in the timetable part, not in the substitutions part below. Here you wrote that if one class has cancelled lesson, the period string in that row is crossed, which is nonesense, cause it's not the point of the issue

Reply:
I was trying to address https://github.com/zorzalo3/zorza/issues/4#issuecomment-1755620648 . However, I have already removed this change